### PR TITLE
Vet ClinicDB: add "join table" for visits

### PR DESCRIPTION
**Pull request by ibr5500**
## In this [PR] I have:

> - Create a table named vets.
> - Create a "join table" called specializations.
> - Create a "join table" called visits.
> - Insert data to the vets table.
> - Insert data to the specialties table.
> - Insert data to the visits table.
> - Write queries to answer the questions.
> - Update README file.

### CODE REVIEW

File: data.sql [hunk 1]
```diff
@@ -62,3 +62,244 @@ UPDATE animals SET owner_id = (
...
+    SELECT id FROM vets
+    WHERE name = 'Stephanie Mendez'
+  )
+);
+
+insert into specializations (species_id, vet_id)
+VALUES (
+  (
+    SELECT id FROM species
+    WHERE name like 'Pokemon'
```
Comment (ShahierNashaat) [inline data.sql]: - [x] Kindly be noticed that there should be `Digimon` instead of `Pokemon`.

File: queries.sql [hunk 2]
```diff
@@ -138,3 +138,68 @@ SELECT owners.full_name, count(*) FROM animals
...
+INNER JOIN visits on animals.id = visits.animals_id
+INNER JOIN vets on vets.id = visits.vet_id
+WHERE vets.name = 'William Tatcher'
+ORDER by date_of_visit DESC LIMIT 1;
+
+-- How many different animals did Stephanie Mendez see?
+SELECT count(*) as differnt_animals FROM animals
+INNER JOIN visits on animals.id = visits.animals_id
+INNER JOIN vets on vets.id = visits.vet_id
+WHERE vets.name = 'Stephanie Mendez';
```
Comment (ShahierNashaat) [inline queries.sql]: - [x] Kindly be noticed that you should use the `DISTINCT` keyword in this query.

```sql
  SELECT DISTINCT animals.name
  FROM visits
  LEFT JOIN animals ON animals.id = visits.animal_id
  LEFT JOIN vets ON vets.id = visits.vet_id
  WHERE vets.name = 'Stephanie Mendez';
```

**Review by ShahierNashaat — CHANGES_REQUESTED** [review]
Hi, @ibr5500 , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by ibr5500** [comment]
Thank you @ShahierNashaat for the code review.

**Review by keroloussamy — APPROVED** [review]
Hi @ibr5500, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## To highlight 🎯

- No linter errors.✔️
- Good code design and implementation. ✔️
- Add all the requirements. ✔️

clap

## Optional suggestions
_Check the comments under the review._

- [ ] x

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by ibr5500** [comment]
Thank you @keroloussamy for the code review.

**Pull request merged**